CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # CHANGELOG
 
+## 2.9.0 (2023-12-05)
+
+MLflow 2.9.0 includes several major features and improvements
+
```

Comment:
Headline block here with a deprecation notice for AI Gateway and the creation of the new deployments API as a replacement. Also, highlight a link to the migration guide. 

Other content to feature at the top:
- The model registry implementation that supports copy_model_version 
- notification for CVE fixes
- single statement about new guides and tutorials

Reply:
sounds good. I'll add them.

Reply:
Added